kp_distance: 0.8
kp_toTarget_angle: 1.3
kp_diff_angle: -0.4
max_v: 0.5
max_w: 1.2
finish_radius: 0.1
rate: 50
path_samples: 500
odom_frame_id : odom
base_link_id : base_link